lms/views/error.py

```diff
@@ -15,31 +16,42 @@ def __init__(self, exc, request):
         self.request = request
 
     @exception_view_config(httpexceptions.HTTPError)
-    @exception_view_config(httpexceptions.HTTPServerError)
```

Comment:
This second view config decorator was pointless as `HTTPServerError` is a subclass of `HTTPError`

Reply:
I was originally thinking that you would want to use the more specific exception here but I guess you're just trying to make it as generic as possible.

Reply:
Yep, just catching all `HTTPError`s here (so all HTTP 4xx and 5xx exceptions deliberately raised by our code) and reporting them to Sentry (they wouldn't be reported normally) and also showing the user the exception message in the error page.